### PR TITLE
[dev] (Select Gap Closure) AB#48476: [G.1.1] - Ability to quick create multiple serving opportunities (one-time event)

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

File: docs/src/inputs/select/examples/exampleRequiredSelect.jsx
Comment (groberts314) [inline docs/src/inputs/select/examples/exampleRequiredSelect.jsx:18]: Should probably be "Select Option" since it is not using the "Users" custom options like some other examples.

File: docs/src/inputs/select/examples/exampleClearableSelect.jsx
Comment (groberts314) [inline docs/src/inputs/select/examples/exampleClearableSelect.jsx:16]: Are the surrounding `<div>`s in all of these examples intentional?

File: src/inputs/select/select.jsx
Comment (groberts314) [inline src/inputs/select/select.jsx:33]: Should say "Select" instead of "Prompt"

File: src/inputs/select/select.jsx
Comment (groberts314) [inline src/inputs/select/select.jsx:27]: Should say "Select" instead of "Prompt"

**Review by groberts314 — CHANGES_REQUESTED** [review]
Looks pretty good to me.  Just a few small fixes and a question.

File: docs/src/inputs/select/examples/exampleLabelSelect.jsx
Comment (fantaJinMode) [inline docs/src/inputs/select/examples/exampleLabelSelect.jsx:10]: NAB: Need to add an empty newline

File: docs/src/inputs/select/examples/exampleRequiredSelect.jsx
Comment (fantaJinMode) [inline docs/src/inputs/select/examples/exampleRequiredSelect.jsx:10]: NAB: Need an empty newline

**Review by fantaJinMode — APPROVED** [review]
LGTM 👍 but agreed with comments of @groberts314

**Review by War777 — CHANGES_REQUESTED** [review]
I found some minor styling issues:

- In the **Searchable Select** example the **No Results Found** label doesn't have a left margin value, it's not aligned to the `<Select />` text.

image

In the **Creatable Select** example:

- The **x**  icon is overlayed above the text.
- The new created option is added with the _Create Option_ text (is that expected)?

image

In the **Creatable Advanced** Select example:

- The custom new option text is behind the **x**  **+** icons.


image

Let me know guys it these might be blocking issues or not, In the meantime I'm marking this as requested changes, if you consider this NABs I'll mark as approved then, thanks a lot.

File: docs/src/inputs/select/examples/exampleClearableSelect.jsx
Comment (cbrewer1689) [reply docs/src/inputs/select/examples/exampleClearableSelect.jsx:16]: Nah.

File: docs/src/inputs/select/examples/exampleRequiredSelect.jsx
Comment (cbrewer1689) [reply docs/src/inputs/select/examples/exampleRequiredSelect.jsx:10]: fixed.

File: docs/src/inputs/select/examples/exampleLabelSelect.jsx
Comment (cbrewer1689) [reply docs/src/inputs/select/examples/exampleLabelSelect.jsx:10]: fixed.

File: src/inputs/select/select.jsx
Comment (cbrewer1689) [reply src/inputs/select/select.jsx:27]: fixed.

File: src/inputs/select/select.jsx
Comment (cbrewer1689) [reply src/inputs/select/select.jsx:33]: fixed.

File: docs/src/inputs/select/examples/exampleClearableSelect.jsx
Comment (cbrewer1689) [reply docs/src/inputs/select/examples/exampleClearableSelect.jsx:16]: fixed.

File: docs/src/inputs/select/examples/exampleRequiredSelect.jsx
Comment (cbrewer1689) [reply docs/src/inputs/select/examples/exampleRequiredSelect.jsx:18]: fixed.

**Comment by cbrewer1689** [comment]
@War777, these are definitely issues that were already existing, but I think it would be good to fix them, i just am not sure if we want to do this now. @sircris, can you weigh in on whether we fix or not? I am thinking maybe like .5-.75 hour of time.

**Comment by sircris** [comment]
@War777 @cbrewer1689 Given the fact that these are existing issues, I can go either way. I want to keep the view of the Sprint Commit in mind. If you don't think this is going to jeopardize that, I'm okay to fix this now. If you think it is, let's create a ticket to deal with this in a future sprint/patch. 

I appreciate the response and consideration for priority. 1 hour of time to address this time is not a problem. Typically, our problem is not the big cut but the "death of a thousand cuts".

Use your best discretion!

**Comment by War777** [comment]
I'm waiting for @cbrewer1689 response to mark it as approved or to wait for the changes if that doesn't affect the sprint planning. Thanks @sircris .

File: docs/src/inputs/select/examples/exampleRequiredSelect.jsx
Comment (groberts314) [reply docs/src/inputs/select/examples/exampleRequiredSelect.jsx]: Resolving.

File: docs/src/inputs/select/examples/exampleClearableSelect.jsx
Comment (groberts314) [reply docs/src/inputs/select/examples/exampleClearableSelect.jsx]: Resolving.

File: src/inputs/select/select.jsx
Comment (groberts314) [reply src/inputs/select/select.jsx]: Resolving.

File: src/inputs/select/select.jsx
Comment (groberts314) [reply src/inputs/select/select.jsx]: Resolving.

**Review by groberts314 — APPROVED** [review]

**Comment by groberts314** [comment]
Re-reviewed and all of my change requests were addressed.

File: docs/src/inputs/select/examples/exampleLabelSelect.jsx
Comment (fantaJinMode) [reply docs/src/inputs/select/examples/exampleLabelSelect.jsx:10]: @cbrewer1689 Could you please check once, I have checked commits but doesn't seems to be fixed ?

**Comment by cbrewer1689** [comment]
@War777 all the styles you mentioned above should be fixed now.

**Review by War777 — APPROVED** [review]
LGTM Thanks @cbrewer1689

**Pull request merged**
